scalameter-core/src/main/scala/org/scalameter/execution/invocation/instrumentation/MethodSignature.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter.execution.invocation.instrumentation
+
+
+/** Contains method signature in form of className.methodName(methodArgs...)
+ *  where methodArgs are written as Java types.
+ *
+ *  Note that it does not contain return type.
+ *  {{{
+ *    // method signature for a method `(range: scala.collection.immutable.Range).by(idx: Int)`
+ *    MethodSignature(scala.collection.immutable.Range, "by", "int").toString == scala.collection.immutable.Range.by(int)
```

Comment:
Ach, my bad. That's right, it should have the following form:
MethodSignature("scala.collection.immutable.Range", "by", "int").toString == "scala.collection.immutable.Range.by(int)"